modules/core/src/lib/layer.ts

```diff
@@ -1118,11 +1126,16 @@ export default abstract class Layer<PropsT extends {} = {}> extends Component<
           useTerrainHeightMap,
           terrainSkipRender
         };
+
         this.setShaderModuleProps({
           // TODO Revisit whether this is necessary once all layers ported to UBO
           shadow: shadowProps,
           terrain: terrainProps,
           layer: {opacity},
+          lighting: lightSources,
```

Comment:
Whenever I see us calling the same thing with different names it makes me ask why... Perhaps we could rename the lighting module?

Reply:
Added to Audit Log